sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -80,13 +80,16 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
 
     createFunction(functions)
 
-    checkAnswer(sql("SHOW functions"), getFunctions("*"))
+    checkAnswer(sql("SHOW functions"), (getFunctions("*") ++
+      Seq(Row("!="), Row("<>"), Row("between"), Row("case"))))
```

Comment:
nit: shall we put this code in `getFunctions`?

Reply:
> nit: shall we put this code in `getFunctions`?

Good ideal, I will try this.